### ISSUE TITLE
Add Angular 9 support

### ISSUE THREAD

**Issue by umutc** [OasisDigital/angular-material-search-select#18]
Hello, i have been facing an error while angular 9 upgrade process of my project.

`Compiling @oasisdigital/angular-material-search-select : es2015 as esm2015
Error: Error on worker #5: Error: Failed to compile entry-point @oasisdigital/angular-material-search-select (es2015 as esm2015) due to compilation errors:
node_modules/@oasisdigital/angular-material-search-select/fesm2015/oasisdigital-angular-material-search-select.js:203:18 - error NG1010: Value at position 1 in the NgModule.imports of SearchSelectModule is not a reference: [object Object]

203         imports: [
                     ~
204             CommonModule,
    ~~~~~~~~~~~~~~~~~~~~~~~~~
... 
211             ReactiveFormsModule
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
212         ],
    ~~~~~~~~~

    at ClusterWorker.compile (/Users/umutcelik/code/magistum/magistum-pwa/node_modules/@angular/compiler-cli/ngcc/src/main.js:177:27)
    at Worker.<anonymous> (/Users/umutcelik/code/magistum/magistum-pwa/node_modules/@angular/compiler-cli/ngcc/src/execution/cluster/worker.js:44:42)
    at Worker.emit (events.js:321:20)
    at process.<anonymous> (internal/cluster/worker.js:32:12)
    at process.emit (events.js:321:20)
    at emit (internal/child_process.js:881:12)
    at processTicksAndRejections (internal/process/task_queues.js:85:21)
    at ClusterMaster.onWorkerMessage (/Users/umutcelik/code/magistum/magistum-pwa/node_modules/@angular/compiler-cli/ngcc/src/execution/cluster/master.js:158:27)
    at /Users/umutcelik/code/magistum/magistum-pwa/node_modules/@angular/compiler-cli/ngcc/src/execution/cluster/master.js:46:95
    at ClusterMaster.<anonymous> (/Users/umutcelik/code/magistum/magistum-pwa/node_modules/@angular/compiler-cli/ngcc/src/execution/cluster/master.js:238:57)
    at step (/Users/umutcelik/code/magistum/magistum-pwa/node_modules/tslib/tslib.js:136:27)
    at Object.next (/Users/umutcelik/code/magistum/magistum-pwa/node_modules/tslib/tslib.js:117:57)
    at /Users/umutcelik/code/magistum/magistum-pwa/node_modules/tslib/tslib.js:110:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/umutcelik/code/magistum/magistum-pwa/node_modules/tslib/tslib.js:106:16)
    at EventEmitter.<anonymous> (/Users/umutcelik/code/magistum/magistum-pwa/node_modules/@angular/compiler-cli/ngcc/src/execution/cluster/master.js:232:32)
    at EventEmitter.emit (events.js:321:20)`

**Comment by kylecordes** [comment]
Thank you. we haven't tried this internally with v9 yet either, I'm sure it will be a modest fix.

**Comment by kylecordes** [comment]
I am in the process of revamping the source code to use the default file arrangement and features of current Angular CLI 9. So not only will it get Angular 9, but also the build etc. will be simplified and 100% off-the-shelf Angular standard.

**Comment by kylecordes** [comment]
I just published version 1.1.1. It upgrades everything to Angular 9, removes a few bespoke build/publish mechanisms in favor of the capability built into Angular CLI.

I didn't have time to test with anything less than the very most current Angular and Angular Material, but if you are moving up to those then you should find it works very well.
